examples/showcase-all-resources.tf

```diff
@@ -0,0 +1,105 @@
+# App
+
+resource "ably-app" "app1" {
+  status                 = "enabled"
+  tlsOnly                = "false"
```

Comment:
Mentioned in the conversation. Use snake case for resource parameters. e.g tls_only

Reply:
Will add shortly.

Reply:
Fixed in 406bb0afce48806dd8f53359b190d2d0a7b46ca2